### ISSUE TITLE
Module path resolution for js & mjs verticle factories fails on windows

### ISSUE THREAD

**Issue by chefhoobajoob** [reactiverse/es4x#556]
See reproducer here, using es4x 0.15.0:
https://github.com/chefhoobajoob/es4x-graal-js-error-reproducer

For some reason, the `mjs` verticle factory is able to load the entry module if it has a `.js` file extension, but otherwise both factories are unable to resolve module paths when launching an es4x module or, if the module does get loaded and includes `require` or `import` statements, to resolve those paths.

**Comment by pmlopes** [comment]
the internal usage of URIs to refer to module paths seem to be incorrect on windows where slashes are mixed and drive leters miss a prefix extra `/`.

Running tests on windows seem to show for example:

```
file://N:\reactiverse\es4x\somefile.js
```

Where it should read:

```
file:///N:/reactiverse/es4x/somefile.js
```

**Comment by chefhoobajoob** [comment]
Is the code that's generating those file uri's with mixed slashes coming from graalvm, or are they under es4x/vertx control?

**Comment by chefhoobajoob** [comment]
oh there's also a leading slash issue that'd be different for an absolute path on windows vs *nix

**Comment by pmlopes** [comment]
This was fixed on 0.16.0
